app/main/views/root.py

```diff
@@ -6,19 +6,34 @@
 from app.authentication.no_token_exception import NoTokenException
 from app.submitter.submitter import Submitter
 from app import settings
+import logging
 import os
 
+decoder = Decoder(settings.EQ_RRM_PUBLIC_KEY, settings.EQ_SR_PRIVATE_KEY, "digitaleq")
 
-with open(settings.EQ_RRM_PUBLIC_KEY, "rb") as public_key_file:
-    rmm_public_key = public_key_file.read()
 
-with open(settings.EQ_SR_PRIVATE_KEY, "rb") as private_key_file:
-    sr_private_key = private_key_file.read()
-
-decoder = Decoder(rmm_public_key, sr_private_key, "digitaleq")
+@main.before_request
+def jwt():
+    if settings.EQ_PRODUCTION:
+        return jwt_decrypt()
+    else:
+        # DEVELOPER mode
+        logging.warning("Developer mode")
+        if request.args.get('token'):
```

Comment:
'token' is used in quite a few places in this file, consider factoring it out into a constant e.g. EQ_URL_QUERY_STRING_JWT_FIELD_NAME